catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/DashboardRepository.java

```diff
@@ -262,6 +263,11 @@ public String getDisplayName() {
       return entity.getDisplayName();
     }
 
+    @Override
```

Comment:
can't these methods moved to EntityRepository, instead of adding to all the extended classes?

Reply:
Doing so, the signature of this method will become more complicated, right?
<img width="903" alt="image" src="https://user-images.githubusercontent.com/32617/153816140-348376df-64d7-4983-9471-840d260bc8c5.png">